spam/spam_tutorial.py

```diff
@@ -97,10 +109,10 @@
 # Don't truncate text fields in the display
 pd.set_option("display.max_colwidth", 0)
 
-df_dev.sample(5, random_state=123)
+df_dev.sample(5, random_state=3)
 
 # %% [markdown]
-# The class distribution varies slightly from class to class, but all are approximately class-balanced.
+# The class distribution varies slightly from class to class, but all are approximately class-balanced. Note that since we don't have labels for the training set, we can't compute class distributions.
```

Comment:
I don't think this line is necessary (unless someone in the feedback session explicitly asked). We can still get a decent approximation of the distribution from the dev set.